### ISSUE TITLE
Usage

### ISSUE THREAD

**Issue by wgmg165** [umang94/large-scale-ml#1]
Hi！is this library has any introduction for usage?

**Comment by umang94** [comment]
@wgmg165 No, There is no introduction for usage. The library is a basic implementation of concepts of perceptron, svm and knn join. The dataset required for each of these are present along with the code. You can check out and run the python scripts. You would need to tweak the file paths for the datasets and install the mrjob python module.
